experiments.go

```diff
@@ -8,11 +8,4 @@ type Experiments struct {
 	Libp2pStreamMounting bool
 	P2pHttpProxy         bool
 	StrategicProviding   bool
-
-	// OverrideSecurityTransports overrides the set of available security
-	// transports when non-empty. This option should eventually migrate some
-	// place more stable.
-	//
-	// Default: ["tls", "secio", "noise"].
-	OverrideSecurityTransports []string `json:",omitempty"`
```

Comment:
The deleted field will just be ignored, right?

Reply:
Also a reminder that we'll need to modify https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md when we land this in go-ipfs

Reply:
> The deleted field will just be ignored, right?

Yep.